source/assets/sass/visual-design/_theme.scss

```diff
@@ -357,8 +363,8 @@ a {
 
 code,
```

Comment:
New card: Because of the way modules get used over and over now, it feels strange to have actual CSS mixed in with variables and functions that we `@use` in most of the other files. I _think_ that should be fine - the CSS should only output the first time this module is used - but probably a good idea to break it out like we normally would.

Reply:
Added this as a checklist item for this card: https://trello.com/c/V2ujZ5Pc/5-migrate-sass-tokens